app/build.gradle.kts

```diff
@@ -14,6 +15,10 @@ if (keystorePropertiesFile.exists()) {
     keystoreProperties.load(FileInputStream(keystorePropertiesFile))
 }
 
+base {
+    archivesName.set("clock")
+}
```

Comment:
why is this needed here while it works the casual way in other apps?

Reply:
It works without this in this app as well, but this is the reason that deprecation warning appears in build. If we want to get rid of it, we will have to do the same in all other apps.

The warning is:
`Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.`

And with full description:
```
The org.gradle.api.plugins.BasePluginConvention type has been deprecated. This is scheduled to be removed in Gradle 9.0. Consult the upgrading guide for further information: https://docs.gradle.org/8.2.1/userguide/upgrading_version_8.html#base_convention_deprecation
        at Build_gradle$1$1.invoke(build.gradle.kts:26)
        (Run with --stacktrace to get the full stack trace of this deprecation warning.)
        at Build_gradle$1.execute(build.gradle.kts:20)
        (Run with --stacktrace to get the full stack trace of this deprecation warning.)
The org.gradle.api.plugins.Convention type has been deprecated. This is scheduled to be removed in Gradle 9.0. Consult the upgrading guide for further information: https://docs.gradle.org/8.2.1/userguide/upgrading_version_8.html#deprecated_access_to_conventions
        at Build_gradle$1$1.invoke(build.gradle.kts:26)
        (Run with --stacktrace to get the full stack trace of this deprecation warning.)
        at Build_gradle$1.execute(build.gradle.kts:17)
        (Run with --stacktrace to get the full stack trace of this deprecation warning.)
```